tests/e2e/sample-attester/install-operator.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+KBS_IMAGE_NAME="${KBS_IMAGE_NAME:-quay.io/confidential-containers/trustee:290fd0eb64ab20f50efbd27cf80542851c0ee17f}"
```

Comment:
Remove the default value for this variable, we want the test to fail if it's not set.

Reply:
We'd need both the trustee and the client images to be specified and they are mandatory for the test to execute. I'll do the changes for the client image customization.
I think the default images have to be provided somehow, otherwise the github action won't be able to execute properly. Where do you reckon these images should be hardcoded?
1) github action: ok for CI but if you run `make test-e2e` the images must be provided in the command
2) Makefile: both CI and `make test-e2e` are ok

Reply:
Makefile